packages/app-backend-vue3/src/components/data.ts

```diff
@@ -371,13 +371,14 @@ function processRefs (instance) {
 function processEventListeners (instance) {
   const emitsDefinition = instance.type.emits
   const declaredEmits = Array.isArray(emitsDefinition) ? emitsDefinition : Object.keys(emitsDefinition ?? {})
+  const normalizedDeclaredEmits = declaredEmits.map(key => kebabize(key))
   const keys = Object.keys(instance.vnode.props ?? {})
   const result = []
   for (const key of keys) {
     const [prefix, ...eventNameParts] = key.split(/(?=[A-Z])/)
     if (prefix === 'on') {
       const eventName = eventNameParts.join('-').toLowerCase()
```

Comment:
This line makes the eventName kebabized.

But the variable [declaredEmits](https://github.com/vuejs/devtools/pull/2051/files#diff-554ccf9aaff1681ee888264aad2cc1a53b7db9271dcf287c655ee6c79b4ea866R373) is a mixed case array, which will cause a mismatch.